src/core/ad-sizes.ts

```diff
@@ -387,7 +387,7 @@ const slotSizeMappings = {
 		mobile: [adSizes.fluid],
 	},
 	'mobile-sticky': {
-		mobile: [adSizes.mobilesticky],
+		mobile: [adSizes.mobilesticky, adSizes.empty],
 	},
 	'crossword-banner-mobile': {
 		mobile: [adSizes.mobilesticky],
```

Comment:
While you're at it do we need it here too? @Amouzle

Reply:
We'll add this in a separate PR